src/hooks/item.ts

```diff
@@ -314,15 +328,15 @@ export default (
           }
 
           return Api.getPublicItemsWithTag({ tagId }, queryConfig).then(
-            (data) => List(data),
+            (data) => convertJs(data),
           );
         },
-        onSuccess: async (items: List<Item>) => {
+        onSuccess: async (items: List<RecordOf<Item>>) => {
           // save items in their own key
           // eslint-disable-next-line no-unused-expressions
           items?.forEach(async (item) => {
             const { id } = item;
-            queryClient.setQueryData(buildItemKey(id), Map(item));
+            queryClient.setQueryData(buildItemKey(id), item);
```

Comment:
Shouldn't this be a record of Item?

Reply:
It is already a Record<Item> (the convertJs function already converts it to Record)